CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ### Changed
 -   Updated styles for the `<Drawer>`.
+-   Updated actionItem prop type for `<Header>`.
```

Comment:
I would list this as an 'Added': Ability to use Avatar in `<Header>` actionItems